homeassistant/components/keba/sensor.py

```diff
@@ -65,7 +75,18 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 class KebaSensor(SensorEntity):
     """The entity class for KEBA charging stations sensors."""
 
-    def __init__(self, keba, key, name, entity_type, icon, unit, device_class=None):
+    def __init__(
+        self,
+        keba,
+        key,
+        name,
+        entity_type,
+        icon,
```

Comment:
May I suggest you use the new SensorEntityDescription, that saves a number of parameters.

Reply:
Yes of course. Could you lead me to a good example which I could adapt?

Reply:
You can e.g. look at components/skybell/sensor.py 
that uses self.entityDescription as well as SensorEntityDescription for setup of default sensor types.

Reply:
Please do use `SensorEntityDescription` as suggested :+1:

Reply:
Done